app/client/views/home/home.html

```diff
@@ -155,11 +199,16 @@ <h1>faq</h1>
 				<div class="page-header">
 					<h1>contact</h1>
 				</div>
-				<div>
-					"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-					Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-					irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-					cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+				<div class="page-text">
+					<ul class="contact-list">
```

Comment:
@qiqi0725 
I tried this on my local. Does this look better as a contact display list?

![image](https://user-images.githubusercontent.com/29107916/46379707-b5491280-c66d-11e8-8f20-dfc95f141bb0.png)

If yes then I'll push it

Reply:
Looks good. Can you add email as well

Reply:
Yes will do